Gems/Atom/Feature/Common/Code/Source/Material/MaterialAssignmentSerializer.cpp

```diff
@@ -80,6 +82,7 @@ namespace AZ
                             LoadAny<AZ::Data::AssetId>(propertyValue, inputPropertyPair.value, context, result) ||
                             LoadAny<AZ::Data::Asset<AZ::Data::AssetData>>(propertyValue, inputPropertyPair.value, context, result) ||
                             LoadAny<AZ::Data::Asset<AZ::RPI::ImageAsset>>(propertyValue, inputPropertyPair.value, context, result) ||
+                            LoadAny<AZ::Data::Asset<AZ::RPI::AttachmentImageAsset>>(propertyValue, inputPropertyPair.value, context, result) ||
```

Comment:
Sorry that I missed this in the original PR. I wish this was handled automatically.